cPP_DSC.adoc

```diff
@@ -956,14 +956,10 @@ NIST SP 800-208 [parameters]
 
 |===
 
-_Application Note {counter:remark_count}_:: _FIPS 186-5 allows the use of SHAKE128 and SHAKE256. Implementations shall use the correct number of bits of output as specified in FIPS 186-5._
+_Application Note {counter:remark_count}_:: _FIPS 186-5 allows the use of SHAKE128 and SHAKE256._
 +
 _DSA may only be used to verify signatures generated prior to the implementation date of FIPS 186-5._
 +
-_The TOE may contain a public key which is integrity protected (e.g., in hardware), in which case the FDP_ITC.1 and FDP_ITC.2 dependencies do not apply. In this case, no dependencies may be chosen._
-+
-_For signature verifications, private keys are not necessary, so there are no dependencies required for generating or destroying cryptographic keys._
```

Comment:
Is this removal warranted because the app note is intended for PP authors not ST authors?